package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "wp-parsely",
+  "version": "2.4.0",
+  "description": "The official WordPress plugin for Parse.ly - makes it a snap to add the required tracking code to enable Parse.ly on your WordPress site.",
+  "main": "build/index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "wp-scripts build",
+    "start": "wp-scripts start",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Parsely/wp-parsely.git"
+  },
+  "author": "",
+  "license": "GPL-2.0-only",
```

This doesn't match what the plugins was already licensed under, which is GPL-2.0-or-later, which is what WP core uses. Keeping it to only 2.0 would technically stop someone using it with a plugin that is only GPL-3.0. 
```suggestion
  "license": "GPL-2.0-or-later",
```